src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -66,7 +66,7 @@
 VIS_LIB_CDN_LINK_TMPL = (
     "https://sdk-bulk-test-endpoint.azureedge.net/bulk-test-details/view/{version}/bulkTestDetails.min.js?version=1"
 )
-VIS_LIB_VERSION = "0.0.32"
+VIS_LIB_VERSION = "0.0.33"
```

Comment:
if this upgrade introduces other features than dotnet support, maybe need an update to CHANGELOG

Reply:
considering you are the release dev who need to take care CHANGELOG in release branch, feel free to do this in this PR or release branch.

Reply:
There is no change to current python flow

Reply:
it's an exact copy of 32 for now and they promise that no impact will happen for python user